Automattic/jetpack
# Like block: Create scaffolding for new dynamic beta Like button block

Resolves https://github.com/Automattic/wp-calypso/issues/84685.

## Proposed changes:

* adds the scaffolding for the new dynamic Like block (beta)

![Markup on 2023-11-28 at 16:32:57](https://github.com/Automattic/jetpack/assets/25105483/c605baf4-3b4b-4fb3-a0b0-10afc7e17ec4)

### Other information:


## Jetpack product discussion

Related project thread: pdDOJh-2JP-p2

## Does this pull request change what data or activity we track or use?

No, it does not.

## Testing instructions:

1. Please check out the branch and build it.
2. Since this new block is in beta, make sure your WordPress site has `define( 'JETPACK_BLOCKS_VARIATION', 'beta' );` line in its `wp-config.php`.
3. Create a new page / post, add the Like block to it and save the changes. The block should be working correctly in backend and frontend. At the moment, the frontend displays just a simple text string ("This is where the like button will go.").
